lib/polyfills.json

```diff
@@ -20,27 +20,10 @@
     "licenses": ["Apache2"],
     "notes": ["A small library that unifies the HTML5 Notifications APIs across different browsers including IE9 & IE10."]
   },
-  "css3pie": {
-    "name": "CSS3 PIE",
-    "authors": ["Jason Johnston"],
-    "href": "http://css3pie.com/",
```

Comment:
that site still exists, why remove it?

Reply:
I haven't checked without deactivating my DNS blocklists but
[https://imgur.com/a/Rkt7Itz](https://imgur.com/a/Rkt7Itz)

Reply:
Okay, I've checked on my computer again and it works but it previously didn't? The Internet, full of mysteries... I'll re-add it